drools-osgi/drools-karaf-features/pom.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.drools</groupId>
+    <artifactId>drools-osgi</artifactId>
+    <version>6.2.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>drools-karaf-features</artifactId>
+
+  <name>Drools :: Karaf Features</name>
+
+  <properties>
+    <!--
+      CONVENTIONS:
+      - All properties used in features.xml are declared here with the prefix "karaf.".
+      - If a property relates to a version property in kie-parent-with-dependencies or jboss-integration-platform-bom,
+        then it must follow the pattern "karaf.[servicemix.]{relatedVersionPropertyName}".
+    -->
+    <!-- TODO we should use the karaf features-maven-plugin -->
+    <!-- Normal reuses -->
+    <karaf.version.org.apache.camel>${version.org.apache.camel}</karaf.version.org.apache.camel>
```

Comment:
I think we should not reference the camel version here. It is not used in the features file.